recipes/stacks/build.sh

```diff
@@ -6,5 +6,7 @@ export CPATH=${PREFIX}/include
  sed -i.bak "41 c extern int encoded_gtypes[4][4];" src/populations.cc
 
 ./configure --prefix=$PREFIX --enable-sparsehash --enable-bam --includedir=${PREFIX}/include/bam --libdir=${PREFIX}/lib
+cp Makefile Makefile.bk
+sed "s|^LIBS=.*$|LIBS=\"-I$PREFIX/include/ -L$PREFIX/lib/ -lz -lgomp\"|" Makefile.bk > Makefile
```

Comment:
you could use `sed -i.bak "s|^LIBS=.*$|LIBS=\"-I$PREFIX/include/ -L$PREFIX/lib/ -lz -lgomp\"|" Makefile`